debug.log

```diff
@@ -0,0 +1,2 @@
+[1116/091808.136:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
+[1118/120811.552:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
This should not be a part of this pull request.
Please remove the file and maybe add `debug.log` to `.gitignore`